ui/imports/shared/controls/TransactionActivityTile.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.15
+import QtQuick.Shapes 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype TransactionActivityTile
+   \inherits StatusListItem
+   \inqmlmodule shared.controls
+   \since shared.controls 1.0
+   \brief It displays list of addresses for wallet activity.
+
+   The \c TransactionActivityTile is base container for tiles used in Wallet activity.
+
+   \qml
+        TransactionActivityTile {
+            title: qsTr("Mainnet Tx hash")
+            width: parent.width
+            roundedCornersBottom: false
+        }
+   \endqml
+*/
+
+StatusListItem {
+    id: root
+
+    /*!
+       \qmlproperty bool TransactionActivityTile::roundedCornersTop
+       This property shows rounded corners at the top of the background rectangle.
+       First tile should always have roundedCorners corners.
```

Comment:
I'm wondering if there is an advantage/requirement for this approach over squared tiles and clip them within a clipped container with rounded corners.

Reply:
Yes. It's because in design only first and second tile has rounded corners.
It might look that it could be solved by having border outside the tile, but issue is hovering. 
When I hover tile I want full fill, so first and last items will have filled color rectangle when hovering with radius on one side of the rectangle.
I think this solution is much better than for example using Shape, especially that it only loads for first and last tile.

![image](https://github.com/status-im/status-desktop/assets/11396062/9e7fb789-f462-41dd-b95e-881305f9f16b)

Reply:
For sure shape won't be an advantage. Still not sure why a simpler approach with the list embedded into the `Rectangle` component won't work; it doesn't worth considering it, though, when you have something in place.

e.g.

```qml
Rectangle {
  radius: 5
  clip: true
  ListView {
    delegate: TransactionActivityTile {}
    }
}

Reply:
@stefandunca and @IvanBelyakoff 
Unfortunately clip doesn't work with radius in QML. I will remove that effect affecting rounded corners as it's API isn't perfect.
Whole solution might look like a hack, but it's best way to handle it pixel-perfect long term and it's proper way to handle such cases.

I will figure out frame component in another issue.

Reply:
@IvanBelyakoff It's because of some optimizations on Qt side.
Generally I decided I will implement it during summary part where I will be using those tiles.